scripts/cti

```diff
@@ -4,16 +4,99 @@
 set -e
 set -o pipefail
 
+cd "$(dirname "$0")"
+
 TAG=""
 PR=""
 WORKSPACE="cluster-test-ci"
-ENV="env"
+ENV=""
 REPORT=""
-MARKER=$USER
+MARKER=$(whoami)
 DEPLOY=yes
 
+K8S=""
+K8S_POOL_SIZE="2"
+K8S_CONTEXT_PATTERN='arn:aws:eks:us-west-2:853397791086:cluster/ct-index-k8s-testnet'
+
+join_args() {
+  retval_join_args=""
+  for var in $*
+  do
+    retval_join_args="${retval_join_args}, \"${var}\""
+  done
+}
+
+join_env_vars() {
+  retval_join_env_vars=""
+  for var in $*
+  do
+    IFS='=' read -ra env_var <<< "$var"
+    retval_join_env_vars="{\"name\":\"${env_var[0]}\", \"value\":\"${env_var[1]}\"}, ${retval_join_env_vars}"
+  done
+}
+
+kube_init_context () {
+  aws eks --region us-west-2 list-clusters &> /dev/null
+  if [[ $? -gt 0 ]]; then
+    echo "aws eks command failed, try awsmfa?"
+    exit 1
+  fi
+  local highest_pool_index=$(($K8S_POOL_SIZE - 1))
+  local context=${K8S_CONTEXT_PATTERN/index/${highest_pool_index}}
+  if kubectl config get-contexts ${context} &> /dev/null; then
+    return
+  fi
+  for ((i = 0; i < ${K8S_POOL_SIZE}; i++)); do
+    aws eks --region us-west-2 update-kubeconfig --name ct-${i}-k8s-testnet
+  done
+}
+
+kube_select_cluster () {
```

Comment:
I think it would be still good to keep ability to set up your own cluster and specify that we want to run on it, instead of auto-selecting shared cluster

Reply:
Updated it, so that `-W` can be set for K8S setup as well.